pytorch_optimizer/optimizer/adahessian.py

```diff
@@ -0,0 +1,164 @@
+import torch
+from torch.optim.optimizer import Optimizer
+
+from pytorch_optimizer.base.exception import NoSparseGradientError
+from pytorch_optimizer.base.optimizer import BaseOptimizer
+from pytorch_optimizer.base.types import BETAS, CLOSURE, DEFAULTS, LOSS, PARAMETERS
+
+# Modified from https://github.com/davda54/ada-hessian/blob/master/ada_hessian.py (MIT David Samuel)
+
+
+class AdaHessian(Optimizer, BaseOptimizer):
```

Comment:
Actually, there was `AdaHessian optimizer` before #59, but I removed it because there were issues with calculating Rademacher distribution and autograd stuff on the recent pytorch (maybe >=1.9).

I guess we need an alternative version that works on the recent pytorch.

Reply:
cool! im looking forward to your work!